templates/documents/general_upload.html

```diff
@@ -33,6 +37,14 @@
 <input type="hidden" name="category_id" value="{$category_id}" />
 <input type="hidden" name="process" value="{$PROCESS}" />
 </form>
+<!-- Drag and drop uploader -->
+
+{php} if ($GLOBALS['document_drag_drop']) { {/php}
+<form method="post" enctype="multipart/form-data" action="{$WEBROOT}/interface/documents/upload.php?patient_id={php}echo $pid{/php}&parent_id={php}echo $category{/php}" class="dropzone">
```

Comment:
those files are referenced in the template file. This is the file that handles the uploading. The js file does not handle the uploading server side.